pkg/docker/helper.go

```diff
@@ -157,12 +158,51 @@ func (r *DockerHelper) Inspect(ids []string, inspectType string, obj interface{}
 	return nil
 }
 
+// FindContainer will try to find a container based on the input labels.
+// If no container is found, it will search for the labels in b64 encording.
+//
+// This plain version of the container find is kept in order to ensure backwards
+// compatibility before the introduction of b64-encoded labels happened.
 func (r *DockerHelper) FindContainer(labels []string) ([]string, error) {
 	args := []string{"ps", "-q", "-a"}
 	for _, label := range labels {
 		args = append(args, "--filter", "label="+label)
 	}
 
+	out, err := r.buildCmd(context.TODO(), args...).Output()
+	if err != nil {
+		labelsB64 := []string{}
+		for _, label := range labels {
+			labelb64 := base64.StdEncoding.EncodeToString([]byte(label))
+			labelsB64 = append(labelsB64, labelb64)
```

Comment:
Rewrite as:
```
labelsB64 = append(labelsB64, base64.StdEncoding.EncodeToString([]byte(label)))
```

Also please move the translation of the labels into the FindContainerB64 function